Templates/TodoList-NodeJS/function.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "table",
```

Comment:
I made all the other comments before this one - I don't think this is a pattern we'd want to lead people to follow.